app/controllers/buyers/applications_controller.rb

```diff
@@ -126,7 +122,7 @@ def resume
   def change_plan
     # there is no need to query available_application_plans as we already have a validation
     service = @cinstance.service
-    new_plan = service.application_plans.stock.find(params[:cinstance][:plan_id])
+    new_plan = accessible_plans.find(plan_id) # .stock
```

Comment:
test pass both with and without stock so I should add a test whatever it should be.
the decision of which kinda depends on https://github.com/3scale/porta/pull/1798/files#r441727410

Reply:
I think it is related to https://github.com/3scale/porta/blob/master/test/integration/user-management-api/buyers_application_plans_test.rb#L113

You cannot buy a custom plan so changing to a custom one does not make sense. As a custom one is tied to only one application

Reply:
Makes sense 👍  I will add a test 😉